connbeat.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Or perhaps even in sts-agent-omnibus, but as `supervisor.conf` is already too here I'm ok with having the connbeat configuration here, too.